modules/virt-configuring-storage-class-bootsource-update.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-configuring-storage-class-bootsource-update_{context}"]
+= Configuring a storage class for custom boot source updates
+
+You must configure a storage class that allows automatic updating for custom boot sources.
+
+.Procedure
+
+* Edit the HyperConverged custom resource (CR) and define a `storageClassName`:
++
+[source,yaml]
+----
+...
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  dataImportCronTemplates:
+  - metadata:
+      name: rhel8-image-cron
+    spec:
+      template:
+        spec:
+          storageClassName: "some-name"
+...
+----
+** Alternatively, you can also set an appropriate storage class name. For example:
++
+[source,terminal]
+----
+oc patch storageclass <current_default_storage_class> -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
```

Comment:
Per our guidelines, use only command per code block. The two `oc patch` commands should be split into separate code blocks, each with its own `[source,terminal]` tag

Reply:
Done.